fixtures/flight/yarn.lock

```diff
@@ -7946,18 +7946,18 @@ react-refresh@^0.11.0:
   integrity sha512-F27qZr8uUqwhWZboondsPx8tnC3Ct3SxZA3V5WyEvujRyyNv0VYPhoBg1gZ8/MV5tubQp76Trw8lTv9hzRBa+A==
 
 react-server-dom-webpack@experimental:
-  version "0.0.0-experimental-6ebfd5b0-20240818"
-  resolved "https://registry.yarnpkg.com/react-server-dom-webpack/-/react-server-dom-webpack-0.0.0-experimental-6ebfd5b0-20240818.tgz#56df6a7a406a033897f9bdd33b649e6e956adcef"
-  integrity sha512-kDPLVKaSKwDpuxGKxWS4w0VEY1O0IUJVksfA39H7nENjmFCuHybZ6rvi+hlXgJcwV3TvVeISLSmf27yvZWUriQ==
+  version "0.0.0-experimental-b3a95caf-20250113"
+  resolved "https://registry.yarnpkg.com/react-server-dom-webpack/-/react-server-dom-webpack-0.0.0-experimental-b3a95caf-20250113.tgz#a2175829be7395acc4d25e0befbd92b233a764c6"
+  integrity sha512-/6/IP8sP/I1CKIBsXvuF3mQEO0Vw/Nu85QyNliCx+K6WT7plTbjpFlEnbJKWoZdh/9uQkG17IZZiGlOgExmjbA==
```

Comment:
Had to regenerate this entry.

We installed an old version of `react-server-dom-webpack` without a dependency on `webpack-sources`.
So Yarn decided to hoist v2 which is not compatible with the `webpack-sources` newer versions of `react-server-dom-webpack` need.

Best fix would be to ditch `predev` in favor of `file:` or `link:` protocols.
`yarn install` in `predev` and `file:` protocols might be the safest option though slower. Will keep an eye out if we encounter these issues more often.